src/components/policy/policy_regular/src/policy_manager_impl.cc

```diff
@@ -213,10 +213,8 @@ bool PolicyManagerImpl::LoadPT(const std::string& file,
       return false;
     }
 
-    if (pt_update->policy_table.module_config.certificate.is_initialized()) {
-      listener_->OnCertificateUpdated(
-          *(pt_update->policy_table.module_config.certificate));
-    }
+    listener_->OnCertificateUpdated(
```

Comment:
@AKalinich-Luxoft What if no certificate in PTU came? You still call OnCetificateUpdated but actualy it is not updated

Reply:
@AByzhynar this was done exactly for handling this case. 
OnCertificateUpdated callback is the trigger for security manager to proceed the handshake. All other listeners can easily handle case when empty certificate arrived. So this callback is mandatory to proceed the handshake, even if no certificate in PTU

Reply:
@AByzhynar BTW the same implementation in external proprietary policy flow